packages/astro/test/fixtures/lit-element/src/components/my-element.js

```diff
@@ -0,0 +1,13 @@
+import { LitElement, html } from 'lit';
+
+export const tagName = 'my-element';
+
+export class MyElement extends LitElement {
+  render() {
+    return html`
+      <div>Testing...</div>
+    `;
+  }
+}
+
+customElements.define('my-element', MyElement);
```

Comment:
You can use the @customElement decorator. It's almost a standard in Lit :)

Reply:
How would that work?

Reply:
```ts
import {LitElement} from "lit";
import {customElement} from "lit/decorators.js";
@customElement("my-element")
class MyElement extends LitElement{
 // ...
}
```
It works the same as customElements.define but its a bit of sugar

Reply:
Yep, I know, but this is just a test.